
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>9. 16-07-2019-Script in python to find the most connected genes to a query gene. :milky_way: &#8212; EXPERIMENTAL JOURNAL PHD</title>
    
  <link href="../../_static/css/theme.css" rel="stylesheet" />
  <link href="../../_static/css/index.c5995385ac14fb8791e8eb36b4908be2.css" rel="stylesheet" />

    
  <link rel="stylesheet"
    href="../../_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../../_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../../_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    
      

    
    <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
    <link rel="stylesheet" href="../../_static/sphinx-book-theme.acff12b8f9c144ce68a297486a2fa670.css" type="text/css" />
    <link rel="stylesheet" type="text/css" href="../../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/tabs.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/panels-main.c949a650a448cc0ae9fd3441c0e17fb0.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/panels-variables.06eb56fa6e07937060861dad626602ad.css" />
    
  <link rel="preload" as="script" href="../../_static/js/index.1c5a1a01449ed65a7b51.js">

    <script id="documentation_options" data-url_root="../../" src="../../_static/documentation_options.js"></script>
    <script src="../../_static/jquery.js"></script>
    <script src="../../_static/underscore.js"></script>
    <script src="../../_static/doctools.js"></script>
    <script src="../../_static/togglebutton.js"></script>
    <script src="../../_static/clipboard.min.js"></script>
    <script src="../../_static/copybutton.js"></script>
    <script async="async" kind="hypothesis" src="https://hypothes.is/embed.js"></script>
    <script kind="utterances">

    var commentsRunWhenDOMLoaded = cb => {
    if (document.readyState != 'loading') {
        cb()
    } else if (document.addEventListener) {
        document.addEventListener('DOMContentLoaded', cb)
    } else {
        document.attachEvent('onreadystatechange', function() {
        if (document.readyState == 'complete') cb()
        })
    }
}

var addUtterances = () => {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src = "https://utteranc.es/client.js";
    script.async = "async";

    script.setAttribute("repo", "leilaicruz/Experimental-journal-jupyterBOOK");
    script.setAttribute("issue-term", "pathname");
    script.setAttribute("theme", "github-light");
    script.setAttribute("label", "ðŸ’¬ comment");
    script.setAttribute("crossorigin", "anonymous");

    sections = document.querySelectorAll("div.section");
    if (sections !== null) {
        section = sections[sections.length-1];
        section.appendChild(script);
    }
}
commentsRunWhenDOMLoaded(addUtterances);
</script>
    <script >var togglebuttonSelector = '.toggle, .admonition.dropdown, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="../../_static/sphinx-book-theme.12a9622fbb08dcb3a2a40b2c02b83a57.js"></script>
    <script async="async" src="https://unpkg.com/thebelab@latest/lib/index.js"></script>
    <script >
        const thebe_selector = ".thebe"
        const thebe_selector_input = "pre"
        const thebe_selector_output = ".output"
    </script>
    <script async="async" src="../../_static/sphinx-thebe.js"></script>
    <link rel="shortcut icon" href="../../_static/flowers.jpg"/>
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="12. Title : 11072019-ylic132-1 and ylic132-2 transformation with the OEP2 :white_check_mark:" href="2019-07-11_Exp_URA-kickout-transformation-ylic132_1_2.html" />
    <link rel="prev" title="8. Title : 15072019-Check of growth in -ade and -ura of 10 clones of Byk832+pbk549 :ok:" href="2019-07-16-Byk832-checking-clones-ade-ura.html" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="en" />
    
  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="80">
    
    <div class="container-fluid" id="banner"></div>

    

    <div class="container-xl">
      <div class="row">
          
<div class="col-12 col-md-3 bd-sidebar site-navigation show" id="site-navigation">
    
        <div class="navbar-brand-box">
    <a class="navbar-brand text-wrap" href="../../index.html">
      
      <img src="../../_static/flowers.jpg" class="logo" alt="logo">
      
      
      <h1 class="site-logo" id="site-title">EXPERIMENTAL JOURNAL PHD</h1>
      
    </a>
</div><form class="bd-search d-flex align-items-center" action="../../search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" >
</form><nav class="bd-links" id="bd-docs-nav" aria-label="Main navigation">
    <div class="bd-toc-item active">
        <ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../../intro.html">
   Experimental progress ðŸ¥¼ ðŸ¦ 
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  General docs
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-01/2021-01-19-Balance-2020.html">
   1. Balance of the 2020
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-07-Balance-of-2019.html">
   2. Balance of the 2019 and outlook to the 2020
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-21-Pandoc-installation.html">
   3. Objective
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-06/2020-06-02-Yearly-Progress-Meeting.html">
   5. Title: Yearly Progress Meeting (Starting my 3rd PhD year)
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Protocols
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../Protocols/Media.html">
   1. YPD media
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../Protocols/Yeast-transformation.html">
   5. Title :  High efficiency Yeast Transformation
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-09-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-09/20092021-Biotek-new-Gal-concentrations-36C.html">
   1. Title : Biotek round on new gal concentrations at 36C
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-09/28092021-Biotek-reduced-volume-to-test-for-supressors.html">
   2. Title : Biotek volume reduced during incubation at 30C to check supressors
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-09/30092021-PCR-checks-to-gal-strains.html">
   3. Title : Genetic checks for the Gal strains used in the Biotek
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-08-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-08/2021-08-01-Biotek-testing-new-dbem3-pGal.html">
   1. Title: Biotek to test ylic140 dbem3dbem1pGal strain
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-08/2021-08-11-transformation_dnrp1dbem1.html">
   2. Title: Transformation/checking on ylic136 and ylic138 with bem1::KANMX
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-08/2021-08-30-Biotek-new-Gal-concentrations.html">
   3. Title: Biotek 30C with new Gal concentration below 0.1%
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-06-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-06/2021-06-03-dbem3-transformation-on-ylic139.html">
   1. Title : Contruction dbem1dbem3+pGal from ylic139 -&gt; ylic140
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-03-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-03/2021-03-01-gDNA-PCR-for-dbem3_2.html">
   1. Title : Genomic extraction and PCR of clean ylic137_2
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-03/2021-03-30-dbem1-transformation-on-pgal-wt.html">
   2. Title : bem1::KanMX transformation in WT+pGal-CDC42
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-02-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-02/2021-02-12-genomic-extractions-and-pcr-ylic137-ylic138.html">
   1. Title : Genomic extractions,PCR for checking and sequencing of dbem3 and dbem3dnrp1 new strains
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-01-2021
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-01/2021-01-25-Checking-dbem3-dbem3dnrp1-new-strains.html">
   1. Title : PCR and sequencing checks for the new strains dbem3 and dbem3dnrp1
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2021-01/2021-01-25-Supp-controls-sfgFP-influence-to-paper.html">
   2. Title: Supp controls on sfgFP influence on the bulk growth
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-12-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-12/2020-12-04-ylic133-ylic136-transformation-with-bem3d.html">
   1. Title: ylic133 and ylic136 transformation with bem3d::NAT
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-12/2020-12-03-Sending-DNA-to-sequence-to-Oxford.html">
   2. Samples
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-11-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-11/2020-11-27-gDNA-extraction-and-PCR-with-ywt04a.html">
   1. Title : gDNA extraction from yWT04a
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-11/2020-11-17-gDNA-extraction-and-PCR-with-yll138.html">
   2. Title : gDNA extraction of yll138
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-10-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-10/2020-10-21-Digestion-Ligation2Sequence.html">
   1. Title : Digestion and Ligation for sequencing
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-10/2020-10-19-Digestion-Ligation-Test-New-SATAYDNA.html">
   2. Title : Digestion and Ligation Test for new SATAY DNA
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-10/2020-10-15-More-DNA-extraction-WT-dnrp1.html">
   3. Title: More DNA extraction of dnrp1 and WT strains
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-10/2020-10-14-bem3d-bem3dnrp1d-strain-construction.html">
   4. Title : ylic133 and ylic136 transformation with bem3d::NAT
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-08-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-08/2020-08-24-Digestion-circularization-PCR.html">
   1. Title: Towards DNA sequencing: Digestion, Circularization and PCR of the transposon-genome junctions.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-08/2020-08-18-DNA-Prep-SATAY.html">
   2. DNA extraction from SATAY samples
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-08/2020-08-10-SATAY-bem2d-nrp1d-WT.html">
   3. Title : 20200810-SATAY for
   <span class="math notranslate nohighlight">
    \(\Delta\)
   </span>
   bem2,
   <span class="math notranslate nohighlight">
    \(\Delta\)
   </span>
   nrp1 and WT
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-07-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-07/2020-07-10-Transformation-of-ylic135-and-ylic136-with-pbK549.html">
   1. Title : Transformation of ylic135 and ylic136 with pbK549 for further SATAY experiment
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-07/2020-07-06-Preparation-for-PCR-check-of-BEM2-bem2.html">
   2. Title : Preparation for PCR check of BEM2 presence and absence in strain ylic135
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-06-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-06/2020-06-22-Essential-and-not-essential-differences-in-interactors.html">
   1. Title : How different are the number of interactors of essential genes compared with non essential genes.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-06/2020-06-17-Experimental-planing.html">
   2. Date
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-04-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-30-Checking-bem2-deletion-on-ylic133_5.html">
   1. Title: Checking bem2 deletion on the transformed strain ylic133_5
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-29-how-commons-properties-are-useful-to-predict-SL.html">
   2. Title : From common properties to Synthetic Lethality prediction
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-24-plan-of-next-experiments.html">
   3. Title: 24042020-Plan for next experiments
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-23-Machine-Learning-study.html">
   4. Title: 23042020-Summary on machine learning study
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-04/2020-04-08-summary-FACs-experiments.html">
   5. Date
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-03-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-03/2020-03-04-FACs.html">
   1. Title : 20200305-FACs experiment with strains ywkd065, ywkd024, ywkd038 and ywkd001
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-02-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-02/2020-02-25-FACs-starting-with-30C-shaking.html">
   1. Title : 20200227-FACs experiment with strains ywkd065, ywkd024, ywkd038 and ywkd001
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-02/2020-02-23-ylic133-5-transformations.html">
   2. Title: Transformation of ylic133_5 with the PCR products from yll137 and yll140
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-02/2020-02-21-hands-on-data-from-facs.html">
   3. Title :What are the output channels from flow cytometry actualling telling us about the fluorescence of every cell?
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-02/2020-02-13-Comparison-of%20Werner-FACs-settings-with-mine.html">
   4. Title : Comparison between the fAcs settings used for Werner in 2017 with the ones I am using currently.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-02/2020-02-09-dnrp1-dbem2-for-satay.html">
   5. Title : Design of the genomic transformation to make dnrp1 and dbem2 strains usable for SATAY
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-01-2020
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-02-04-FACs-as-microscopy-conditions-prep-III.html">
   1. Title: FACs experiment with the same conditions as the microscopy done to quantify the cell sizes for the pAl1-Cdc42-sfGFP strains (III).
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-28-FACs-as-microscopy-conditions-prep-II.html">
   2. Title: FACs experiment with the same conditions as the microscopy done to quantify the cell sizes for the pGAl1-Cdc42-sfGFP strains (II).
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-20-FACs-as-microscopy-conditions-prep.html">
   3. Title: FACs experiment with the same conditions as the microscopy done to quantify the cell sizes for the pAl1-Cdc42-sfGFP strains. (I)
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-07-machine-learning-proposal-projects.html">
   4. Title: Machine learning projects proposal for MsC/BsC/internships
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2020-01/2020-01-14-essential-genes-insights.html">
   5. Title:14012020-Mapping essential genes  in WT and getting insights from it.
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-10-2019
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-10/2019-10-09-Sanity-check-ylic133%2BpBk549.html">
   1. Title: 09102019- Sanity check of transformants ylic133+pBk549, Byk832+pBk549 to know if they suitable for SATAY. :smile:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-10/2019-10-03-pBK549-transformation-in-ylic133-clones-III.html">
   2. Title: 03102019- III pBK549 transformation on ylic133 for further sanity check in SATAY :blush: :punch:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-10/2019-10-01-pBK549-transformation-in-ylic133-clonesII.html">
   3. Title : 01102019- II pBK549 transformation on ylic133 for further sanity check in SATAY :pensive:
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-09-2019
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-09/2019-09-24-ylic134-mating-yEK7a.html">
   1. Title :
   <font color="#ee4466">
    24092019-Mating of ylic134 with yEK7a :broken_heart:
   </font>
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-09/2019-09-24-pBK549-transformation-in-ylic133-and-Byk832.html">
   2. Title : 24092019- pBK549 transformation on ylic133 for further sanity check in SATAY. :pensive:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-09/2019-09-12-ylic133_2%263-transformation-pLL112.html">
   3. Title : 12092019-Transformation of ylic133 with the pLL112 plasmid :pensive:
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-08-2019
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-27-Sporulation-protocol-ylic133%2ByEK7a.html">
   1. Title: 27082019-Sporulation protocol of 8 re-streakings from ylic133_5+yEk7a :pensive:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-22-Exp_Mating-liquid-ylic133_5-yEK7a.html">
   2. Title:
   <font color="#ee4466">
    22082019 Mating using liquid drops of yEK7a and ylic133_5 :broken_heart:
   </font>
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-16-some-plots-gene_int-data.html">
   3. Title : 16082019 Script for the visualization of the correlation between number of common interactions and type of interactions
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-12-Checking-clones-ura-and-ade-growth.html">
   4. Title : 12082019-Checking clones of Byk832 and ylic33_1 for SATAY :ok:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-07-Exp-crossing-plate-yEK7a-and-ylic133_1.html">
   5. Title :
   <font color="#ee4466">
    07082019-Crossing of yEK7a with ylic133_1 :broken_heart: :pensive:
   </font>
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-08/2019-08-05_Exp-Byk832-transformation-with-pBK549.html">
   6. Title: 05082019-Byk832 transformation with pBk549 :ok: :white_check_mark:
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-07-2019
 </span>
</p>
<ul class="current nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="2019-07-31-Extra-checking-steps-SATAY-Byk832T.html">
   1. Title: 31072019 Double check Byk832T biolog.  replicates suitability for SATAY.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-07-24-Strategy-to-SATAY-target-strains.html">
   2. Title : 24072019-Strategies to generate the strains I want to study using SATAY :page_with_curl:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-07-23-SATAY-Byk832-1st-attempt.html">
   4. Title: 29072019-SATAY 1st attempt with Byk832-T :pray: :confused: :x:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-07-19-glycerol-stocks-ylic133-bio-replicates.html">
   5. Title : 23072019-Glycerols stocks of ylic133 biological replicates :snowflake:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-07-19_Sequencing-ylic133-biol-replicates.html">
   6. Title: 22072019-Sequencing of ylic133 biological replicates :ok: :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-07-17_Exp-Genomic-prep-ylic133-clones.html">
   7. Title : 17072019-DNA extraction of ylic133: ura3- ade2- clones anad PCR :ok: :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-07-16-Byk832-checking-clones-ade-ura.html">
   8. Title : 15072019-Check of growth in -ade and -ura of 10 clones of Byk832+pbk549 :ok:
  </a>
 </li>
 <li class="toctree-l1 current active">
  <a class="current reference internal" href="#">
   9. 16-07-2019-Script in python to find the most connected genes to a query gene. :milky_way:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-07-11_Exp_URA-kickout-transformation-ylic132_1_2.html">
   12. Title : 11072019-ylic132-1 and ylic132-2  transformation with the OEP2 :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-07-10_Byk832-transformation-pBk549.html">
   14. Title : 10072019-Byk832 transformation using pBK549 plasmid (:ok:)
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-07-08-Data-ylic132_biolog-replicates-sequencing.html">
   15. Title: 26062019-Sequencing of the URA integration in ade2 locus in ylic132 :ok: :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="2019-07-04_Exp_pBK549_digestion_plasmid.html">
   16. Title : 03072019-Digestion pattern of pBK549 (Succesful) :ok:
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-06-2019
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-28_Exp_URA_kickout_transf.html">
   1. Title: 26062019-ylic132  transformation with the OEP2 (unsuccesful) :x: :disappointed:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-26_Data_Sequencing_ylic132.html">
   2. Title: 26062019-Sequencing of the URA integration in ade2 locus in ylic132 :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-25_SATAY_card.html">
   3. Title: 25062019-Experimental Card about the SATAY experiment
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-20_Exp_Byk832_transformation_pbk549.html">
   4. Title: 19062019-Byk832 (ETH strain) transformation with pbK549 (very low effciency) :pensive: :confused:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-17_Exp_URA_kickout_transf.html">
   5. Title: 14062019-ylic132  transformation with the OEP2 (unsuccesful)  :x: :disappointed:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-06_Exp_URA_Transformation.html">
   6. Title: 06062019-Yeast transformation with the OEP1 :white_check_mark: :white_check_mark:
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-03_Exp_new%20primers_ade2_locus.html">
   7. Title: 03062019-Design of new primers, for the right homology arm, for transformation that avoid the overlap with the RGA1 gene.
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-06/2019-06-03_Data_Sequencing_ade2_yll3a.html">
   8. Title: 03062019-Sequencing the homology arms of yll3a strain around the ade2 gene :ok:.
  </a>
 </li>
</ul>
<p class="caption">
 <span class="caption-text">
  Experimental journal-05-2019
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../2019-05/2019-05-20_Exp_URA_TRANSFORMATION.html">
   1. Title: URA transformation in ADE2 locus
  </a>
 </li>
</ul>

    </div>
</nav> <!-- To handle the deprecated key -->

<div class="navbar_extra_footer">
  Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
</div>

</div>


          


          
<main class="col py-md-3 pl-md-4 bd-content overflow-auto" role="main">
    
    <div class="topbar container-xl fixed-top">
    <div class="topbar-contents row">
        <div class="col-12 col-md-3 bd-topbar-whitespace site-navigation show"></div>
        <div class="col pl-md-4 topbar-main">
            
            <button id="navbar-toggler" class="navbar-toggler ml-0" type="button" data-toggle="collapse"
                data-toggle="tooltip" data-placement="bottom" data-target=".site-navigation" aria-controls="navbar-menu"
                aria-expanded="true" aria-label="Toggle navigation" aria-controls="site-navigation"
                title="Toggle navigation" data-toggle="tooltip" data-placement="left">
                <i class="fas fa-bars"></i>
                <i class="fas fa-arrow-left"></i>
                <i class="fas fa-arrow-up"></i>
            </button>
            
            
<div class="dropdown-buttons-trigger">
    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn" aria-label="Download this page"><i
            class="fas fa-download"></i></button>

    <div class="dropdown-buttons">
        <!-- ipynb file if we had a myst markdown file -->
        
        <!-- Download raw file -->
        <a class="dropdown-buttons" href="../../_sources/journal/2019-07/2019-07-16_common-interactor-genes-workflow.md"><button type="button"
                class="btn btn-secondary topbarbtn" title="Download source file" data-toggle="tooltip"
                data-placement="left">.md</button></a>
        <!-- Download PDF via print -->
        <button type="button" id="download-print" class="btn btn-secondary topbarbtn" title="Print to PDF"
            onClick="window.print()" data-toggle="tooltip" data-placement="left">.pdf</button>
    </div>
</div>

            <!-- Source interaction buttons -->

<div class="dropdown-buttons-trigger">
    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn"
        aria-label="Connect with source repository"><i class="fab fa-github"></i></button>
    <div class="dropdown-buttons sourcebuttons">
        <a class="repository-button"
            href="https://github.com/leilaicruz/Experimental-journal-jupyterBOOK"><button type="button" class="btn btn-secondary topbarbtn"
                data-toggle="tooltip" data-placement="left" title="Source repository"><i
                    class="fab fa-github"></i>repository</button></a>
        <a class="issues-button"
            href="https://github.com/leilaicruz/Experimental-journal-jupyterBOOK/issues/new?title=Issue%20on%20page%20%2Fjournal/2019-07/2019-07-16_common-interactor-genes-workflow.html&body=Your%20issue%20content%20here."><button
                type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip" data-placement="left"
                title="Open an issue"><i class="fas fa-lightbulb"></i>open issue</button></a>
        
    </div>
</div>

            <!-- Full screen (wrap in <a> to have style consistency -->

<a class="full-screen-button"><button type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip"
        data-placement="bottom" onclick="toggleFullScreen()" aria-label="Fullscreen mode"
        title="Fullscreen mode"><i
            class="fas fa-expand"></i></button></a>

            <!-- Launch buttons -->

        </div>

        <!-- Table of contents -->
        <div class="d-none d-md-block col-md-2 bd-toc show">
            
            <div class="tocsection onthispage pt-5 pb-3">
                <i class="fas fa-list"></i> Contents
            </div>
            <nav id="bd-toc-nav">
                <ul class="visible nav section-nav flex-column">
 <li class="toc-h1 nav-item toc-entry">
  <a class="reference internal nav-link" href="#">
   9. 16-07-2019-Script in python to find the most connected genes to a query gene. :milky_way:
  </a>
  <ul class="visible nav section-nav flex-column">
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#goal">
     9.1. Goal
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h1 nav-item toc-entry">
  <a class="reference internal nav-link" href="#python-code">
   10. Python code
  </a>
 </li>
 <li class="toc-h1 nav-item toc-entry">
  <a class="reference internal nav-link" href="#results-figure">
   11. Results Figure
  </a>
 </li>
</ul>

            </nav>
        </div>
    </div>
</div>
    <div id="main-content" class="row">
        <div class="col-12 col-md-9 pl-md-3 pr-md-0">
        
              <div>
                
  <div class="section" id="script-in-python-to-find-the-most-connected-genes-to-a-query-gene-milky-way">
<h1><span class="section-number">9. </span>16-07-2019-Script in python to find the most connected genes to a query gene. :milky_way:<a class="headerlink" href="#script-in-python-to-find-the-most-connected-genes-to-a-query-gene-milky-way" title="Permalink to this headline">Â¶</a></h1>
<div class="section" id="goal">
<h2><span class="section-number">9.1. </span>Goal<a class="headerlink" href="#goal" title="Permalink to this headline">Â¶</a></h2>
<p>From the physical and genetic interactors of a specific gene, I am interested in finding which of the interactors of the interactors are also interactors of the initial query gene.</p>
<p>This calculation can maybe triggers hypothesis about how  is the correlation in the amount of connectivity of one gene to another gene and the type of interactions they are more prone to share.</p>
<p>Perhaps, it is like this:
<img alt="" src="../../_images/Model_cartoon-05.png" /></p>
<p>The following picture depicts what I am looking for in the <a class="reference external" href="https://www.yeastgenome.org/">SGD database</a> for every gene of interest:
<img alt="" src="../../_images/cartoon-for-common-genes-workflow07.png" /></p>
<p>The following python code depicts what I have done, that can be reused and improved  for other purposes.</p>
</div>
</div>
<div class="section" id="python-code">
<h1><span class="section-number">10. </span>Python code<a class="headerlink" href="#python-code" title="Permalink to this headline">Â¶</a></h1>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="kn">import</span> <span class="nn">pandas</span> <span class="k">as</span> <span class="nn">pd</span>
<span class="kn">import</span> <span class="nn">numpy</span> <span class="k">as</span> <span class="nn">np</span>
<span class="kn">from</span> <span class="nn">collections</span> <span class="kn">import</span> <span class="n">defaultdict</span>
<span class="kn">import</span> <span class="nn">seaborn</span> <span class="k">as</span> <span class="nn">sns</span>
<span class="kn">import</span> <span class="nn">matplotlib.pyplot</span> <span class="k">as</span> <span class="nn">plt</span>
<span class="o">%</span><span class="n">matplotlib</span> <span class="n">inline</span>


<span class="c1">## good website to study dataframes</span>
<span class="c1">#https://www.shanelynn.ie/using-pandas-dataframe-creating-editing-viewing-data-in-python/</span>
</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="n">data</span><span class="o">=</span><span class="n">pd</span><span class="o">.</span><span class="n">read_excel</span><span class="p">(</span><span class="sa">r</span><span class="s1">&#39;C:\Users\linigodelacruz\Documents\PhD_2018\Documentation\Calculations\data_sgd\Interaction_data_sgd_downloads.xlsx&#39;</span><span class="p">,</span><span class="n">header</span><span class="o">=</span><span class="mi">17</span><span class="p">,</span><span class="n">encoding</span><span class="o">=</span><span class="s2">&quot;utf-8-sig&quot;</span><span class="p">)</span>
<span class="n">col_label</span><span class="o">=</span><span class="n">data</span><span class="o">.</span><span class="n">columns</span><span class="o">.</span><span class="n">values</span>
<span class="n">data_go</span><span class="o">=</span><span class="n">pd</span><span class="o">.</span><span class="n">read_excel</span><span class="p">(</span><span class="sa">r</span><span class="s1">&#39;C:\Users\linigodelacruz\Documents\PhD_2018\Documentation\Calculations\data_sgd\slim_goterms_data_sgd_downloads.xlsx&#39;</span><span class="p">,</span><span class="n">header</span><span class="o">=</span><span class="mi">14</span><span class="p">,</span><span class="n">encoding</span><span class="o">=</span><span class="s2">&quot;utf-8-sig&quot;</span><span class="p">)</span>
<span class="n">col_label_go</span><span class="o">=</span><span class="n">data_go</span><span class="o">.</span><span class="n">columns</span><span class="o">.</span><span class="n">values</span>
<span class="n">data_go</span><span class="o">.</span><span class="n">set_index</span><span class="p">(</span><span class="n">col_label_go</span><span class="p">[</span><span class="mi">1</span><span class="p">],</span><span class="n">inplace</span><span class="o">=</span><span class="kc">True</span><span class="p">)</span>
</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span>


<span class="n">d2</span> <span class="o">=</span> <span class="n">defaultdict</span><span class="p">(</span><span class="nb">dict</span><span class="p">)</span>
<span class="n">query</span><span class="o">=</span><span class="p">[</span><span class="s1">&#39;BEM1&#39;</span><span class="p">]</span> <span class="c1"># here you can put the gene of interest</span>
<span class="c1"># giant for loop</span>
<span class="n">names1</span> <span class="o">=</span> <span class="n">query</span>
<span class="n">i</span><span class="o">=-</span><span class="mi">1</span>
<span class="k">for</span> <span class="n">query1</span> <span class="ow">in</span> <span class="n">names1</span><span class="p">:</span>
    <span class="c1">#filtering the table just for the value of the query</span>
    <span class="n">q1</span> <span class="o">=</span> <span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Standard_Gene_Name_(Bait)&#39;</span><span class="p">]</span><span class="o">==</span><span class="n">query1</span><span class="p">]</span>
    <span class="n">q1_interact</span><span class="o">=</span><span class="n">q1</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">3</span><span class="p">]]</span><span class="o">.</span><span class="n">unique</span><span class="p">()</span>
   <span class="c1"># a for loop for all the interactors of query</span>
    <span class="k">for</span> <span class="n">query2</span> <span class="ow">in</span> <span class="n">q1_interact</span><span class="p">:</span>


        <span class="n">q2</span><span class="o">=</span><span class="n">data</span><span class="p">[</span><span class="n">data</span><span class="p">[</span><span class="s1">&#39;Standard_Gene_Name_(Bait)&#39;</span><span class="p">]</span><span class="o">==</span><span class="n">query2</span><span class="p">]</span> <span class="c1">#these are get_query(q1[i])</span>
        <span class="n">q2_interact</span><span class="o">=</span><span class="n">q2</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">3</span><span class="p">]]</span><span class="o">.</span><span class="n">unique</span><span class="p">()</span>
        <span class="n">d</span> <span class="o">=</span> <span class="n">defaultdict</span><span class="p">(</span><span class="nb">int</span><span class="p">)</span>
        <span class="n">common</span> <span class="o">=</span> <span class="p">[]</span>

        <span class="k">for</span> <span class="n">name1</span>  <span class="ow">in</span> <span class="n">q2_interact</span><span class="p">:</span>
            <span class="k">if</span> <span class="n">name1</span> <span class="ow">in</span> <span class="n">q1_interact</span><span class="p">:</span> <span class="c1"># if a gene interactor of the query1 is in interactors of query 2</span>
                <span class="n">common</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">name1</span><span class="p">)</span>
                <span class="n">d</span><span class="p">[</span><span class="n">name1</span><span class="p">]</span> <span class="o">+=</span> <span class="mi">1</span>


        <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;common&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="n">common</span>
        <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span><span class="n">query2</span><span class="p">][</span><span class="s2">&quot;names of genes&quot;</span><span class="p">]</span><span class="o">=</span><span class="n">query2</span>
        <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;n_common&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">common</span><span class="p">)</span>
        <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;len_i_1&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">q1</span><span class="p">)</span>
        <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;len_i_2&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">q2</span><span class="p">)</span>
        <span class="k">if</span> <span class="nb">len</span><span class="p">(</span><span class="n">q1</span><span class="p">)</span><span class="o">==</span><span class="mi">0</span><span class="p">:</span>
            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;</span><span class="si">% o</span><span class="s2">f query subset&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="mi">0</span>
        <span class="k">else</span><span class="p">:</span>
            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;</span><span class="si">% o</span><span class="s2">f query subset&quot;</span><span class="p">]</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">d</span><span class="p">)</span><span class="o">/</span><span class="nb">len</span><span class="p">(</span><span class="n">q1_interact</span><span class="p">)</span> <span class="o">*</span><span class="mi">100</span>

        <span class="k">if</span> <span class="nb">len</span><span class="p">(</span><span class="n">q2</span><span class="p">)</span><span class="o">==</span><span class="mi">0</span><span class="p">:</span>
            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;</span><span class="si">% o</span><span class="s2">f query 2 subset  &quot;</span><span class="p">]</span> <span class="o">=</span> <span class="mi">0</span>
        <span class="k">else</span><span class="p">:</span>
            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span> <span class="n">query2</span><span class="p">][</span><span class="s2">&quot;</span><span class="si">% o</span><span class="s2">f query 2 subset  &quot;</span><span class="p">]</span> <span class="o">=</span> <span class="nb">len</span><span class="p">(</span><span class="n">d</span><span class="p">)</span><span class="o">/</span><span class="nb">len</span><span class="p">(</span><span class="n">q2_interact</span><span class="p">)</span> <span class="o">*</span><span class="mi">100</span>

            <span class="n">q1_filt</span><span class="o">=</span><span class="n">q1</span><span class="p">[</span><span class="n">q1</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">3</span><span class="p">]]</span><span class="o">==</span><span class="n">query2</span><span class="p">]</span>
            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span><span class="n">query2</span><span class="p">][</span><span class="s2">&quot;interact_annotation&quot;</span><span class="p">]</span><span class="o">=</span><span class="n">q1_filt</span><span class="p">[</span><span class="n">col_label</span><span class="p">[</span><span class="mi">4</span><span class="p">]]</span>
            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span><span class="n">query2</span><span class="p">][</span><span class="s1">&#39;GO_slim_query&#39;</span><span class="p">]</span><span class="o">=</span> <span class="n">data_go</span><span class="o">.</span><span class="n">loc</span><span class="p">[</span><span class="n">query1</span><span class="p">][</span><span class="n">col_label_go</span><span class="p">[</span><span class="mi">4</span><span class="p">]]</span>
            <span class="c1">#d2[query1,query2][&#39;GO_slim_interactors&#39;]= data_go[data_go[col_label_go[1]]==query2][col_label_go[4]]</span>

            <span class="n">d2</span><span class="p">[</span><span class="n">query1</span><span class="p">,</span><span class="n">query2</span><span class="p">][</span><span class="s1">&#39;GO_slim_interactors&#39;</span><span class="p">]</span><span class="o">=</span> <span class="n">data_go</span><span class="o">.</span><span class="n">loc</span><span class="p">[</span><span class="n">query2</span><span class="p">][</span><span class="n">col_label_go</span><span class="p">[</span><span class="mi">4</span><span class="p">]]</span>


</pre></div>
</div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="n">df</span><span class="o">=</span><span class="n">pd</span><span class="o">.</span><span class="n">DataFrame</span><span class="p">(</span><span class="n">d2</span><span class="p">)</span><span class="o">.</span><span class="n">T</span>
<span class="n">df</span>
</pre></div>
</div>
<div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }
<div class="highlight-none notranslate"><div class="highlight"><pre><span></span>.dataframe tbody tr th {
    vertical-align: top;
}

.dataframe thead th {
    text-align: right;
}
</pre></div>
</div>
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th></th>
      <th>% of query 2 subset</th>
      <th>% of query subset</th>
      <th>GO_slim_interactors</th>
      <th>GO_slim_query</th>
      <th>common</th>
      <th>interact_annotation</th>
      <th>len_i_1</th>
      <th>len_i_2</th>
      <th>n_common</th>
      <th>names of genes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th rowspan="61" valign="top">BEM1</th>
      <th>ATS1</th>
      <td>15.2542</td>
      <td>2.80374</td>
      <td>Gene (optional)
ATS1                    cytopl...</td>
      <td>Gene (optional)
BEM1                    cellul...</td>
      <td>[RPS8A, BEM1, STE50, RVS161, SAC3, ROM2, TPM1,...</td>
      <td>4872    Negative Genetic
Name: Experiment_Type...</td>
      <td>415</td>
      <td>66</td>
      <td>9</td>
      <td>ATS1</td>
    </tr>
    <tr>
      <th>PMT2</th>
      <td>23.6364</td>
      <td>8.09969</td>
      <td>Gene (optional)
PMT2                          ...</td>
      <td>Gene (optional)
BEM1                    cellul...</td>
      <td>[PEP1, ECM33, PMT1, RGP1, ERD1, GET2, GLO3, PM...</td>
      <td>5487    Negative Genetic
Name: Experiment_Type...</td>
      <td>415</td>
      <td>143</td>
      <td>26</td>
      <td>PMT2</td>
    </tr>
    <tr>
      <th>LTE1</th>
      <td>13.8264</td>
      <td>13.3956</td>
      <td>Gene (optional)
LTE1                          ...</td>
      <td>Gene (optional)
BEM1                    cellul...</td>
      <td>[CDC24, SWD1, FUS3, UBP14, BEM1, MBP1, BRE1, S...</td>
      <td>5830    Negative Genetic
Name: Experiment_Type...</td>
      <td>415</td>
      <td>367</td>
      <td>43</td>
      <td>LTE1</td>
    </tr>
    <tr>
      <th>SNC1</th>
      <td>44</td>
      <td>3.42679</td>
      <td>Gene (optional)
SNC1             Golgi apparat...</td>
      <td>Gene (optional)
BEM1                    cellul...</td>
      <td>[SEC17, SEC18, CYK3, ARF1, SWF1, VPS1, ELO3, S...</td>
      <td>7080    Affinity Capture-Western
Name: Experim...</td>
      <td>415</td>
      <td>50</td>
      <td>11</td>
      <td>SNC1</td>
    </tr>
    <tr>
      <th>CLN3</th>
      <td>35.7143</td>
      <td>7.78816</td>
      <td>Gene (optional)
CLN3                          ...</td>
      <td>Gene (optional)
BEM1                    cellul...</td>
      <td>[STE50, MBP1, STE5, RPA14, MNN10, SSD1, IPK1, ...</td>
      <td>7971    Positive Genetic
Name: Experiment_Type...</td>
      <td>415</td>
      <td>106</td>
      <td>25</td>
      <td>CLN3</td>
    </tr>
    <tr>
      <th>CDC24</th>
      <td>27.4809</td>
      <td>11.215</td>
      <td>Gene (optional)
CDC24                         ...</td>
      <td>Gene (optional)
BEM1                    cellul...</td>
      <td>[CDC24, BOI1, BEM1, RDI1, STE5, GIC2, RGA2, SA...</td>
      <td>8231         Synthetic Lethality
8232    Affin...</td>
      <td>415</td>
      <td>200</td>
      <td>36</td>
      <td>CDC24</td>
    </tr>
    <tr>
      <th>SWD1</th>
      <td>18.8811</td>
      <td>8.41121</td>
      <td>Gene (optional)
SWD1                          ...</td>
      <td>Gene (optional)
BEM1                    cellul...</td>
      <td>[RXT2, STE50, BRE1, RPA14, SAC3, MSN5, SEM1, V...</td>
      <td>10815    Positive Genetic
Name: Experiment_Typ...</td>
      <td>415</td>
      <td>200</td>
      <td>27</td>
      <td>SWD1</td>
    </tr>
    <tr>
      <th>BUD14</th>
      <td>26.7442</td>
      <td>7.16511</td>
      <td>Gene (optional)
      BUD14                         ...</td>
            <td>Gene (optional)
      BEM1                    cellul...</td>
            <td>[RXT2, RRP7, STE50, TPS2, MTH1, IPK1, SWI4, BE...</td>
            <td>11634    Positive Genetic
      Name: Experiment_Typ...</td>
            <td>415</td>
            <td>99</td>
            <td>23</td>
            <td>BUD14</td>
          </tr>
          <tr>
            <th>PDR3</th>
            <td>17.5</td>
            <td>2.18069</td>
            <td>Gene (optional)
      PDR3                          ...</td>
            <td>Gene (optional)
      BEM1                    cellul...</td>
            <td>[MNN10, SEM1, SWI4, BEM2, MNN11, OPI3, ELM1]</td>
            <td>13524    Two-hybrid
      Name: Experiment_Type_(Req...</td>
            <td>415</td>
            <td>41</td>
            <td>7</td>
            <td>PDR3</td>
          </tr>
          <tr>
            <th>FUS3</th>
            <td>23.4177</td>
            <td>11.5265</td>
            <td>Gene (optional)
      FUS3                          ...</td>
            <td>Gene (optional)
      BEM1                    cellul...</td>
            <td>[CLN3, BUD14, FUS3, BOI1, RXT2, BEM1, STE50, S...</td>
            <td>15220    Affinity Capture-Western
      Name: Experi...</td>
            <td>415</td>
            <td>249</td>
            <td>37</td>
            <td>FUS3</td>
          </tr>
          <tr>
            <th>PEP1</th>
            <td>15.7895</td>
            <td>1.86916</td>
            <td>Gene (optional)
      PEP1        Golgi apparatus
      PE...</td>
            <td>Gene (optional)
      BEM1                    cellul...</td>
            <td>[PEP1, PMR1, RIM101, RIC1, RIM21, VPS5]</td>
            <td>15656    Negative Genetic
      Name: Experiment_Typ...</td>
            <td>415</td>
            <td>45</td>
            <td>6</td>
            <td>PEP1</td>
          </tr>
          <tr>
            <th>SEC17</th>
            <td>23.7624</td>
            <td>7.47664</td>
            <td>Gene (optional)
      SEC17                    cytop...</td>
            <td>Gene (optional)
      BEM1                    cellul...</td>
            <td>[SEC18, SWF1, MSN5, GET2, VAM7, IST3, CNB1, YK...</td>
            <td>19834    Co-fractionation
      Name: Experiment_Typ...</td>
            <td>415</td>
            <td>114</td>
            <td>24</td>
            <td>SEC17</td>
          </tr>
          <tr>
            <th>SKT5</th>
            <td>33.6364</td>
            <td>11.5265</td>
            <td>Gene (optional)
      SKT5                          ...</td>
            <td>Gene (optional)
      BEM1                    cellul...</td>
            <td>[RPS8A, CHS3, ECM33, BEM1, STE50, RVS161, CYK3...</td>
            <td>20873    Negative Genetic
      Name: Experiment_Typ...</td>
            <td>415</td>
            <td>169</td>
            <td>37</td>
            <td>SKT5</td>
          </tr>
          <tr>
            <th>RPS8A</th>
            <td>0</td>
            <td>0</td>
            <td>Gene (optional)
      RPS8A                         ...</td>
            <td>Gene (optional)
      BEM1                    cellul...</td>
            <td>[]</td>
            <td>21501    Negative Genetic
      Name: Experiment_Typ...</td>
            <td>415</td>
            <td>5</td>
            <td>0</td>
            <td>RPS8A</td>
          </tr>
          <tr>
            <th>BOI1</th>
            <td>27.2727</td>
            <td>4.6729</td>
            <td>Gene (optional)
      BOI1                cellular b...</td>
            <td>Gene (optional)
      BEM1                    cellul...</td>
            <td>[EXO84, BEM1, RRP7, MTH1, SEC3, BOI2, BEM2, CD...</td>
            <td>22819       Negative Genetic
      22820    Affinity...</td>
            <td>415</td>
            <td>78</td>
            <td>15</td>
            <td>BOI1</td>
          </tr>
          <tr>
            <th>NTH2</th>
            <td>16.6667</td>
            <td>0.934579</td>
            <td>Gene (optional)
            NTH2                          ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[BEM1, STE50, TPS2]</td>
                  <td>24979    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>20</td>
                  <td>3</td>
                  <td>NTH2</td>
                </tr>
                <tr>
                  <th>MNN2</th>
                  <td>36.6197</td>
                  <td>8.09969</td>
                  <td>Gene (optional)
            MNN2                          ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[PMT2, MNN2, VAM6, PMT1, ARF1, ERD1, GET2, SEC...</td>
                  <td>27420    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>82</td>
                  <td>26</td>
                  <td>MNN2</td>
                </tr>
                <tr>
                  <th>CHS3</th>
                  <td>30.9735</td>
                  <td>10.9034</td>
                  <td>Gene (optional)
            CHS3                          ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[CDC24, SKT5, RPS8A, MNN2, CHS3, VAM6, ARF1, M...</td>
                  <td>27890    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>182</td>
                  <td>35</td>
                  <td>CHS3</td>
                </tr>
                <tr>
                  <th>RPS11B</th>
                  <td>7.69231</td>
                  <td>0.623053</td>
                  <td>Gene (optional)
            RPS11B                        ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[BEM1, ARF1]</td>
                  <td>30303    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>26</td>
                  <td>2</td>
                  <td>RPS11B</td>
                </tr>
                <tr>
                  <th>UBP14</th>
                  <td>22.9885</td>
                  <td>6.23053</td>
                  <td>Gene (optional)
            UBP14                         ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[CYC8, BEM1, STE50, RVS161, MNN10, LSM6, RVS16...</td>
                  <td>30960    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>92</td>
                  <td>20</td>
                  <td>UBP14</td>
                </tr>
                <tr>
                  <th>ECM33</th>
                  <td>30.5389</td>
                  <td>15.8879</td>
                  <td>Gene (optional)
            ECM33                         ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[PMT2, PEP1, SKT5, BEM1, STE50, MBP1, VAM6, PM...</td>
                  <td>32618    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>184</td>
                  <td>51</td>
                  <td>ECM33</td>
                </tr>
                <tr>
                  <th>SEC18</th>
                  <td>16.25</td>
                  <td>4.04984</td>
                  <td>Gene (optional)
            SEC18             Golgi appara...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[SEC18, SSD1, LAS21, PBS2, RPS21B, CNB1, YKT6,...</td>
                  <td>33169    Co-fractionation
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>85</td>
                  <td>13</td>
                  <td>SEC18</td>
                </tr>
                <tr>
                  <th>RXT2</th>
                  <td>24.1935</td>
                  <td>9.34579</td>
                  <td>Gene (optional)
            RXT2                          ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[LTE1, FUS3, STE50, RVS161, SAC3, SSD1, SEM1, ...</td>
                  <td>35601    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>174</td>
                  <td>30</td>
                  <td>RXT2</td>
                </tr>
                <tr>
                  <th>EXO84</th>
                  <td>37.931</td>
                  <td>3.42679</td>
                  <td>Gene (optional)
            EXO84                   cell c...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[EXO84, SEC5, SEC3, SEC4, SEC15, SEC6, EXO70, ...</td>
                  <td>36506    Co-purification
            Name: Experiment_Type...</td>
                  <td>415</td>
                  <td>56</td>
                  <td>11</td>
                  <td>EXO84</td>
                </tr>
                <tr>
                  <th>CYC8</th>
                  <td>11.2903</td>
                  <td>2.18069</td>
                  <td>Gene (optional)
            CYC8                          ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[CYC8, SAC3, SKN7, SWI6, ROM2, YDJ1, SIN3]</td>
                  <td>38572    Two-hybrid
            Name: Experiment_Type_(Req...</td>
                  <td>415</td>
                  <td>78</td>
                  <td>7</td>
                  <td>CYC8</td>
                </tr>
                <tr>
                  <th>BEM1</th>
                  <td>100</td>
                  <td>100</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[ATS1, PMT2, LTE1, SNC1, CLN3, CDC24, SWD1, BU...</td>
                  <td>48372    PCA
            Name: Experiment_Type_(Required) ...</td>
                  <td>415</td>
                  <td>415</td>
                  <td>321</td>
                  <td>BEM1</td>
                </tr>
                <tr>
                  <th>LDB16</th>
                  <td>0</td>
                  <td>0</td>
                  <td>NaN</td>
                  <td>NaN</td>
                  <td>[]</td>
                  <td>NaN</td>
                  <td>415</td>
                  <td>0</td>
                  <td>0</td>
                  <td>LDB16</td>
                </tr>
                <tr>
                  <th>RRP7</th>
                  <td>4.16667</td>
                  <td>0.623053</td>
                  <td>Gene (optional)
            RRP7                          ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[RPS27B, RIC1]</td>
                  <td>48377    Two-hybrid
            Name: Experiment_Type_(Req...</td>
                  <td>415</td>
                  <td>51</td>
                  <td>2</td>
                  <td>RRP7</td>
                </tr>
                <tr>
                  <th>STE50</th>
                  <td>12.3711</td>
                  <td>3.73832</td>
                  <td>Gene (optional)
            STE50                     cyto...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[STE50, VAM6, STE5, PSP1, PTP3, VAM7, KEL2, ST...</td>
                  <td>48378    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>127</td>
                  <td>12</td>
                  <td>STE50</td>
                </tr>
                <tr>
                  <th>RVS161</th>
                  <td>35.1724</td>
                  <td>15.8879</td>
                  <td>Gene (optional)
            RVS161                     cel...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[CLN3, BUD14, SKT5, MNN2, CHS3, RXT2, BEM1, CY...</td>
                  <td>48381    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>262</td>
                  <td>51</td>
                  <td>RVS161</td>
                </tr>
                <tr>
                  <th>...</th>
                  <td>...</td>
                  <td>...</td>
                  <td>...</td>
                  <td>...</td>
                  <td>...</td>
                  <td>...</td>
                  <td>...</td>
                  <td>...</td>
                  <td>...</td>
                  <td>...</td>
                </tr>
                <tr>
                  <th>RPS10A</th>
                  <td>0</td>
                  <td>0</td>
                  <td>Gene (optional)
            RPS10A                        ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[]</td>
                  <td>49007    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>4</td>
                  <td>0</td>
                  <td>RPS10A</td>
                </tr>
                <tr>
                  <th>SNC2</th>
                  <td>43.5897</td>
                  <td>5.29595</td>
                  <td>Gene (optional)
            SNC2            Golgi apparatu...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[SNC1, SEC17, EXO84, SEC5, SEC3, SEC15, SEC6, ...</td>
                  <td>49010    Affinity Capture-Western
            Name: Experi...</td>
                  <td>415</td>
                  <td>59</td>
                  <td>17</td>
                  <td>SNC2</td>
                </tr>
                <tr>
                  <th>SVL3</th>
                  <td>20</td>
                  <td>3.73832</td>
                  <td>Gene (optional)
            SVL3                cellular b...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[BEM1, RPS16B, OCA6, BEM2, MNN11, MOG1, SRL3, ...</td>
                  <td>49014    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>69</td>
                  <td>12</td>
                  <td>SVL3</td>
                </tr>
                <tr>
                  <th>MET31</th>
                  <td>20</td>
                  <td>0.934579</td>
                  <td>Gene (optional)
            MET31                         ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[STE50, RVS161, YDJ1]</td>
                  <td>49015    Two-hybrid
            Name: Experiment_Type_(Req...</td>
                  <td>415</td>
                  <td>20</td>
                  <td>3</td>
                  <td>MET31</td>
                </tr>
                <tr>
                  <th>VPS16</th>
                  <td>35</td>
                  <td>2.18069</td>
                  <td>Gene (optional)
            VPS16                         ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[VAM6, SEC4, VAM7, NYV1, VPS33, YPT7, VAM3]</td>
                  <td>49016    Co-fractionation
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>32</td>
                  <td>7</td>
                  <td>VPS16</td>
                </tr>
                <tr>
                  <th>SGF11</th>
                  <td>19.3103</td>
                  <td>8.72274</td>
                  <td>Gene (optional)
            SGF11                         ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[LTE1, RXT2, BEM1, STE50, BRE1, CYK3, SAC3, SE...</td>
                  <td>49017    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>252</td>
                  <td>28</td>
                  <td>SGF11</td>
                </tr>
                <tr>
                  <th>RPL21B</th>
                  <td>12</td>
                  <td>0.934579</td>
                  <td>Gene (optional)
            RPL21B                        ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[BEM1, PAT1, GAS1]</td>
                  <td>49021    Positive Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>25</td>
                  <td>3</td>
                  <td>RPL21B</td>
                </tr>
                <tr>
                  <th>ELP3</th>
                  <td>21.608</td>
                  <td>13.3956</td>
                  <td>Gene (optional)
            ELP3                          ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[ATS1, RPS8A, RXT2, BEM1, RRP7, STE50, RVS161,...</td>
                  <td>49022    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>295</td>
                  <td>43</td>
                  <td>ELP3</td>
                </tr>
                <tr>
                  <th>ELP4</th>
                  <td>24.6032</td>
                  <td>9.65732</td>
                  <td>Gene (optional)
            ELP4                          ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[BEM1, STE50, RVS161, BRE1, UBP1, GET2, SWI4, ...</td>
                  <td>49027    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>163</td>
                  <td>31</td>
                  <td>ELP4</td>
                </tr>
                <tr>
                  <th>BEM3</th>
                  <td>50</td>
                  <td>4.98442</td>
                  <td>Gene (optional)
            BEM3                  cell cor...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[CDC24, BEM1, RVS161, RVS167, BEM2, SLT2, BCK1...</td>
                  <td>49028    Positive Genetic
            49029    Positive Ge...</td>
                  <td>415</td>
                  <td>40</td>
                  <td>16</td>
                  <td>BEM3</td>
                </tr>
                <tr>
                  <th>RNY1</th>
                  <td>25</td>
                  <td>1.55763</td>
                  <td>Gene (optional)
            RNY1                cytoplasm
            ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[BEM1, SPT3, GIM4, RPS21B, ELP3]</td>
                  <td>49033    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>20</td>
                  <td>5</td>
                  <td>RNY1</td>
                </tr>
                <tr>
                  <th>KES1</th>
                  <td>27.5229</td>
                  <td>9.34579</td>
                  <td>Gene (optional)
            KES1            Golgi apparatu...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[BUD14, BEM1, STE50, VAM6, ARF1, SAC3, SAC7, P...</td>
                  <td>49034    Negative Genetic
            49037    Negative Ge...</td>
                  <td>415</td>
                  <td>126</td>
                  <td>30</td>
                  <td>KES1</td>
                </tr>
                <tr>
                  <th>YPL150W</th>
                  <td>0</td>
                  <td>0</td>
                  <td>NaN</td>
                  <td>NaN</td>
                  <td>[]</td>
                  <td>NaN</td>
                  <td>415</td>
                  <td>0</td>
                  <td>0</td>
                  <td>YPL150W</td>
                </tr>
                <tr>
                  <th>RRD2</th>
                  <td>31.0345</td>
                  <td>2.80374</td>
                  <td>Gene (optional)
            RRD2                         o...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[SAC3, SSD1, GIM4, CDC55, SLT2, RTT101, PBS2, ...</td>
                  <td>49041    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>32</td>
                  <td>9</td>
                  <td>RRD2</td>
                </tr>
                <tr>
                  <th>BEM4</th>
                  <td>45</td>
                  <td>2.80374</td>
                  <td>Gene (optional)
            BEM4                    cytopl...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[CDC24, BEM1, BEM2, KSS1, CDC42, STE11, ROM2, ...</td>
                  <td>49044    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>25</td>
                  <td>9</td>
                  <td>BEM4</td>
                </tr>
                <tr>
                  <th>SVS1</th>
                  <td>0</td>
                  <td>0</td>
                  <td>NaN</td>
                  <td>NaN</td>
                  <td>[]</td>
                  <td>NaN</td>
                  <td>415</td>
                  <td>0</td>
                  <td>0</td>
                  <td>SVS1</td>
                </tr>
                <tr>
                  <th>CBC2</th>
                  <td>8.13397</td>
                  <td>5.29595</td>
                  <td>Gene (optional)
            CBC2            nucleus
            CBC2  ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[RPS8A, PAT1, BRE1, SAC3, SSD1, SLX9, IST3, MO...</td>
                  <td>49047    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>369</td>
                  <td>17</td>
                  <td>CBC2</td>
                </tr>
                <tr>
                  <th>PPQ1</th>
                  <td>6.25</td>
                  <td>0.311526</td>
                  <td>Gene (optional)
            PPQ1                    cytopl...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[ELP3]</td>
                  <td>49048    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>17</td>
                  <td>1</td>
                  <td>PPQ1</td>
                </tr>
                <tr>
                  <th>TCO89</th>
                  <td>20</td>
                  <td>7.16511</td>
                  <td>Gene (optional)
            TCO89                         ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[RPS8A, VAM6, RDI1, RPA14, SSD1, PPM1, GET2, A...</td>
                  <td>49049    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>129</td>
                  <td>23</td>
                  <td>TCO89</td>
                </tr>
                <tr>
                  <th>SSO1</th>
                  <td>25</td>
                  <td>2.49221</td>
                  <td>Gene (optional)
            SSO1               cytoplasm
            S...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[SNC1, SEC17, ARF1, VAM7, YKT6, NYV1, SSO2, SNC2]</td>
                  <td>49051    Affinity Capture-Western
            Name: Experi...</td>
                  <td>415</td>
                  <td>61</td>
                  <td>8</td>
                  <td>SSO1</td>
                </tr>
                <tr>
                  <th>YAR1</th>
                  <td>19.2308</td>
                  <td>1.55763</td>
                  <td>Gene (optional)
            YAR1                          ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[BEM1, ELP2, IKI3, ELP6, NST1]</td>
                  <td>49053    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>33</td>
                  <td>5</td>
                  <td>YAR1</td>
                </tr>
                <tr>
                  <th>CLN2</th>
                  <td>30.1724</td>
                  <td>10.9034</td>
                  <td>Gene (optional)
            CLN2                          ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[CLN3, SKT5, CHS3, BEM1, RVS161, PAT1, VAM6, C...</td>
                  <td>49056    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>177</td>
                  <td>35</td>
                  <td>CLN2</td>
                </tr>
                <tr>
                  <th>EAF3</th>
                  <td>11.7647</td>
                  <td>6.23053</td>
                  <td>Gene (optional)
            EAF3                          ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[BRE1, RVS167, SPT3, VPS72, GIM4, RIM101, APQ1...</td>
                  <td>49060    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>241</td>
                  <td>20</td>
                  <td>EAF3</td>
                </tr>
                <tr>
                  <th>SEC8</th>
                  <td>39.2857</td>
                  <td>3.42679</td>
                  <td>Gene (optional)
            SEC8                 cell cort...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[EXO84, SEC5, SEM1, RVS167, SEC3, SEC4, SEC15,...</td>
                  <td>49062             Co-purification
            49063    Aff...</td>
                  <td>415</td>
                  <td>80</td>
                  <td>11</td>
                  <td>SEC8</td>
                </tr>
                <tr>
                  <th>SPE3</th>
                  <td>0</td>
                  <td>0</td>
                  <td>Gene (optional)
            SPE3                          ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[]</td>
                  <td>49064    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>6</td>
                  <td>0</td>
                  <td>SPE3</td>
                </tr>
                <tr>
                  <th>NVJ2</th>
                  <td>0</td>
                  <td>0</td>
                  <td>Gene (optional)
            NVJ2                cytoplasm
            ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[]</td>
                  <td>49065    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>2</td>
                  <td>0</td>
                  <td>NVJ2</td>
                </tr>
                <tr>
                  <th>SYT1</th>
                  <td>26.3158</td>
                  <td>1.55763</td>
                  <td>Gene (optional)
            SYT1                          ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[RGP1, RVS167, ISC1, RIC1, CDC42]</td>
                  <td>49066    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>20</td>
                  <td>5</td>
                  <td>SYT1</td>
                </tr>
                <tr>
                  <th>CLB5</th>
                  <td>10.2362</td>
                  <td>4.04984</td>
                  <td>Gene (optional)
            CLB5                          ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[PAT1, MSN5, VPS72, SWI4, CDC55, RIM101, SLT2,...</td>
                  <td>49067    Negative Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>196</td>
                  <td>13</td>
                  <td>CLB5</td>
                </tr>
                <tr>
                  <th>RHO1</th>
                  <td>20</td>
                  <td>3.42679</td>
                  <td>Gene (optional)
            RHO1                         G...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[RDI1, SAC7, SEC3, SLT2, SKN7, ROM2, ZDS1, BNI...</td>
                  <td>49068    Co-fractionation
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>78</td>
                  <td>11</td>
                  <td>RHO1</td>
                </tr>
                <tr>
                  <th>SKI3</th>
                  <td>12.1951</td>
                  <td>3.11526</td>
                  <td>Gene (optional)
            SKI3                cytoplasm
            ...</td>
                  <td>Gene (optional)
            BEM1                    cellul...</td>
                  <td>[RPS8A, PAT1, SAC3, MNN10, RVS167, LSM1, RPL8B...</td>
                  <td>49069    Positive Genetic
            Name: Experiment_Typ...</td>
                  <td>415</td>
                  <td>104</td>
                  <td>10</td>
                  <td>SKI3</td>
                </tr>
              </tbody>
            </table>
            <p>321 rows Ã— 10 columns</p>
      </div>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span>
<span class="c1"># I dont want the query to be in the plot</span>
<span class="n">df</span><span class="o">=</span><span class="n">pd</span><span class="o">.</span><span class="n">DataFrame</span><span class="p">(</span><span class="n">d2</span><span class="p">)</span><span class="o">.</span><span class="n">T</span>
<span class="n">a</span><span class="o">=</span><span class="n">df</span>
<span class="n">a_col</span><span class="o">=</span><span class="n">a</span><span class="o">.</span><span class="n">columns</span><span class="o">.</span><span class="n">values</span>
<span class="c1">#0&#39;% of query 2 subset  &#39;,</span>
<span class="c1">#1&#39;% of query subset&#39;,</span>
<span class="c1">#2&#39;GO_slim_interactors&#39;</span>
<span class="c1">#3&#39;GO_slim_query&#39;</span>
<span class="c1">#4&#39;common&#39;,</span>
<span class="c1">#5&#39;interact_annotation&#39;,</span>
<span class="c1">#6&#39;len_i_1&#39;,</span>
<span class="c1">#7&#39;len_i_2&#39;,</span>
<span class="c1">#8&#39;n_common&#39;,</span>
<span class="c1">#9&#39;names of genes&#39;</span>
<span class="n">ab</span><span class="o">=</span><span class="n">a</span><span class="p">[</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query subset&#39;</span><span class="p">][::</span><span class="o">-</span><span class="mi">1</span><span class="p">]</span>
<span class="n">pos</span><span class="o">=</span> <span class="n">np</span><span class="o">.</span><span class="n">arange</span><span class="p">(</span><span class="nb">len</span><span class="p">(</span><span class="n">a</span><span class="o">.</span><span class="n">iloc</span><span class="p">[:,</span><span class="mi">4</span><span class="p">]))</span>
<span class="c1">#making a data frame of both datasets for better handling them</span>
<span class="c1">#ab_df=pd.DataFrame({&#39;% of query subset&#39;:ab,&#39;names of genes&#39;:names2[::-1]})</span>
<span class="c1">#sorting the data by the values</span>
<span class="n">absorted</span><span class="o">=</span><span class="n">a</span><span class="o">.</span><span class="n">sort_values</span><span class="p">(</span><span class="n">by</span><span class="o">=</span><span class="p">[</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query subset&#39;</span><span class="p">])</span>

<span class="n">genes_to_plot</span><span class="o">=</span><span class="n">absorted</span><span class="p">[</span><span class="s1">&#39;names of genes&#39;</span><span class="p">]</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="nb">len</span><span class="p">(</span><span class="n">a</span><span class="o">.</span><span class="n">iloc</span><span class="p">[:,</span><span class="mi">4</span><span class="p">])</span><span class="o">-</span><span class="mi">20</span><span class="p">:</span><span class="nb">len</span><span class="p">(</span><span class="n">a</span><span class="o">.</span><span class="n">iloc</span><span class="p">[:,</span><span class="mi">4</span><span class="p">])]</span>
<span class="n">numbers_to_plot</span><span class="o">=</span><span class="n">absorted</span><span class="p">[</span><span class="s1">&#39;</span><span class="si">% o</span><span class="s1">f query subset&#39;</span><span class="p">]</span><span class="o">.</span><span class="n">iloc</span><span class="p">[</span><span class="nb">len</span><span class="p">(</span><span class="n">a</span><span class="o">.</span><span class="n">iloc</span><span class="p">[:,</span><span class="mi">4</span><span class="p">])</span><span class="o">-</span><span class="mi">20</span><span class="p">:</span><span class="nb">len</span><span class="p">(</span><span class="n">a</span><span class="o">.</span><span class="n">iloc</span><span class="p">[:,</span><span class="mi">4</span><span class="p">])]</span>

<span class="n">fig</span><span class="p">,</span> <span class="n">ax</span> <span class="o">=</span> <span class="n">plt</span><span class="o">.</span><span class="n">subplots</span><span class="p">(</span><span class="n">figsize</span><span class="o">=</span><span class="p">(</span><span class="mi">10</span><span class="p">,</span><span class="mi">10</span><span class="p">))</span>         <span class="c1"># Sample figsize in inches</span>
<span class="n">plt</span><span class="o">.</span><span class="n">barh</span><span class="p">(</span><span class="n">pos</span><span class="p">[</span><span class="mi">0</span><span class="p">:</span><span class="mi">20</span><span class="p">],</span><span class="n">numbers_to_plot</span><span class="p">,</span><span class="n">align</span><span class="o">=</span><span class="s1">&#39;edge&#39;</span><span class="p">,</span><span class="n">tick_label</span><span class="o">=</span><span class="n">genes_to_plot</span><span class="p">,</span><span class="n">color</span><span class="o">=</span><span class="p">(</span><span class="mf">0.2</span><span class="p">,</span> <span class="mf">0.4</span><span class="p">,</span> <span class="mf">0.6</span><span class="p">,</span> <span class="mf">0.6</span><span class="p">))</span>

<span class="n">ax</span><span class="o">.</span><span class="n">tick_params</span><span class="p">(</span><span class="n">labelbottom</span><span class="o">=</span><span class="s1">&#39;on&#39;</span><span class="p">,</span><span class="n">labeltop</span><span class="o">=</span><span class="s1">&#39;on&#39;</span><span class="p">)</span>
<span class="n">ax</span><span class="o">.</span><span class="n">grid</span><span class="p">(</span><span class="n">color</span><span class="o">=</span><span class="s1">&#39;k&#39;</span><span class="p">,</span> <span class="n">linestyle</span><span class="o">=</span><span class="s1">&#39;-&#39;</span><span class="p">,</span> <span class="n">linewidth</span><span class="o">=</span><span class="mf">0.5</span><span class="p">)</span>

<span class="n">ax</span><span class="o">.</span><span class="n">text</span><span class="p">(</span><span class="mi">120</span><span class="p">,</span><span class="mi">22</span><span class="p">,</span><span class="s2">&quot;Top_most_connected_gene&quot;</span><span class="p">,</span><span class="n">fontsize</span><span class="o">=</span><span class="mi">17</span><span class="p">)</span>
<span class="n">ax</span><span class="o">.</span><span class="n">text</span><span class="p">(</span><span class="mi">120</span><span class="p">,</span> <span class="mi">0</span><span class="p">,</span>  <span class="n">data_go</span><span class="o">.</span><span class="n">loc</span><span class="p">[</span><span class="n">genes_to_plot</span><span class="p">[</span><span class="mi">18</span><span class="p">]][</span><span class="n">col_label_go</span><span class="p">[</span><span class="mi">4</span><span class="p">]],</span> <span class="n">fontsize</span><span class="o">=</span><span class="mi">15</span><span class="p">)</span>
<span class="n">ax</span><span class="o">.</span><span class="n">text</span><span class="p">(</span><span class="mi">120</span><span class="p">,</span><span class="o">-</span><span class="mi">3</span><span class="p">,</span><span class="s2">&quot;interaction_annotation&quot;</span><span class="p">,</span><span class="n">fontsize</span><span class="o">=</span><span class="mi">17</span><span class="p">)</span>
<span class="n">ax</span><span class="o">.</span><span class="n">text</span><span class="p">(</span><span class="mi">120</span><span class="p">,</span> <span class="o">-</span><span class="mi">6</span><span class="p">,</span>  <span class="n">df</span><span class="o">.</span><span class="n">loc</span><span class="p">[</span><span class="n">query</span><span class="p">[</span><span class="mi">0</span><span class="p">],</span><span class="n">genes_to_plot</span><span class="p">[</span><span class="mi">18</span><span class="p">]][</span><span class="n">a_col</span><span class="p">[</span><span class="mi">5</span><span class="p">]],</span> <span class="n">fontsize</span><span class="o">=</span><span class="mi">15</span><span class="p">)</span>
<span class="n">ax</span><span class="o">.</span><span class="n">text</span><span class="p">(</span><span class="mi">196</span><span class="p">,</span><span class="mi">22</span><span class="p">,</span><span class="s2">&quot;Query_gene&quot;</span><span class="p">,</span><span class="n">fontsize</span><span class="o">=</span><span class="mi">17</span><span class="p">)</span>
<span class="n">ax</span><span class="o">.</span><span class="n">text</span><span class="p">(</span><span class="mi">196</span><span class="p">,</span> <span class="mi">0</span><span class="p">,</span>  <span class="n">data_go</span><span class="o">.</span><span class="n">loc</span><span class="p">[</span><span class="n">query</span><span class="p">][</span><span class="n">col_label_go</span><span class="p">[</span><span class="mi">4</span><span class="p">]],</span> <span class="n">fontsize</span><span class="o">=</span><span class="mi">15</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xticks</span><span class="p">(</span><span class="n">fontsize</span><span class="o">=</span><span class="mi">18</span><span class="p">,</span> <span class="n">rotation</span><span class="o">=</span><span class="mi">0</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">yticks</span><span class="p">(</span><span class="n">fontsize</span><span class="o">=</span><span class="mi">18</span><span class="p">,</span> <span class="n">rotation</span><span class="o">=</span><span class="mi">0</span><span class="p">)</span>
<span class="n">plt</span><span class="o">.</span><span class="n">xlabel</span><span class="p">(</span><span class="s1">&#39;Percentage of the conection of interactors with_&#39;</span> <span class="o">+</span> <span class="s2">&quot;&quot;</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">query</span><span class="p">),</span><span class="n">fontsize</span><span class="o">=</span><span class="mi">18</span><span class="p">)</span>

<span class="n">plt</span><span class="o">.</span><span class="n">savefig</span><span class="p">(</span><span class="s2">&quot;common_interactors_&quot;</span> <span class="o">+</span> <span class="s2">&quot;&quot;</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">query</span><span class="p">)</span> <span class="o">+</span> <span class="s2">&quot;.svg&quot;</span><span class="p">,</span><span class="n">dpi</span><span class="o">=</span><span class="mi">300</span><span class="p">,</span><span class="nb">format</span><span class="o">=</span><span class="s1">&#39;svg&#39;</span><span class="p">)</span>
</pre></div>
</div>
</div>
<div class="section" id="results-figure">
<h1><span class="section-number">11. </span>Results Figure<a class="headerlink" href="#results-figure" title="Permalink to this headline">Â¶</a></h1>
<p><img alt="Figure-showing-the-results" src="../../_images/output_common-genes-workflow.png" /></p>
</div>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            kernelName: "python3",
            path: "./journal/2019-07"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

              </div>
              
        
        <div class='prev-next-bottom'>
            
    <a class='left-prev' id="prev-link" href="2019-07-16-Byk832-checking-clones-ade-ura.html" title="previous page"><span class="section-number">8. </span>Title : 15072019-Check of growth in -ade and -ura of 10 clones of Byk832+pbk549 :ok:</a>
    <a class='right-next' id="next-link" href="2019-07-11_Exp_URA-kickout-transformation-ylic132_1_2.html" title="next page"><span class="section-number">12. </span>Title : 11072019-ylic132-1 and ylic132-2  transformation with the OEP2 :white_check_mark:</a>

        </div>
        
        </div>
    </div>
    <footer class="footer mt-5 mt-md-0">
    <div class="container">
      <p>
        
          By Leila IÃ±igo de la Cruz<br/>
        
            &copy; Copyright Creative Commons.<br/>
      </p>
    </div>
  </footer>
</main>


      </div>
    </div>
  
  <script src="../../_static/js/index.1c5a1a01449ed65a7b51.js"></script>

  
  </body>
</html>